src/org/graphstream/algorithm/LongestPath.java

```diff
@@ -129,6 +131,11 @@ public Path getLongestPath() {
         }
         return path;
     }
+    
+    @Result
+    public String defaultResult() {
```

Comment:
Why is the default result always a String? Shouldnt it be a graph or a collection of nodes?

Reply:
It's true that @Result annotation can be confusing. I created it with the meaning "Default result display". That's why you can find some algorithms with just a function "defaultMessage()" to inform users that the result was stored in attributes.